SQA_AT/src/main/java/com/tricentis/demowebshop/pageobject/WebShopMainPage.java

```diff
@@ -46,7 +48,7 @@ public void logout() {
 
     public List<String> getComputersGroupListName() {
         List<String> list = computersGroupList.stream()
-                .map(f -> f.getAttribute("href"))
+                .map(f -> f.getAttribute(HREF_ATTR))
                 .collect(Collectors.toList());
         list = list.stream()
                 .map(g -> g.substring(g.indexOf("com/") + 4))
```

Comment:
we can do like this:

        return computersGroupList.stream()
                .map(f -> f.getAttribute(HREF_ATTR))
                .map(g -> g.substring(g.indexOf("com/") + 4))
                .collect(Collectors.toList());

Reply:
Pls avoid             Thread.sleep(2000) int this class